### PR TITLE
feat: use usd in exchanges table

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Closes https://app.clickup.com/t/861mruapk

Since we can only retrieve USD from coingecko, im defaulting the currency to USD, see comments of pr for fetching details --> https://github.com/ArdentHQ/arkscan/pull/237

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/views/components/tables/desktop/exchanges.blade.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
             <x-tables.headers.desktop.text name="general.exchange.top_pairs" />
 
             <x-tables.headers.desktop.number name="general.exchange.price">
-                <span>({{ Settings::currency()}})</span>
+                <span>(USD)</span>
```
Comment (alexbarnsley) [inline resources/views/components/tables/desktop/exchanges.blade.php:10]: Can this use the lang for now?

```suggestion
                <span>(@lang('currencies.usd.currency'))</span>
```

File: resources/views/components/tables/desktop/exchanges.blade.php [hunk 2]
```diff
@@ -16,7 +16,7 @@ class="text-right"
                 breakpoint="md-lg"
                 responsive
             >
-                <span>({{ Settings::currency()}})</span>
+                <span>(USD)</span>
```
Comment (alexbarnsley) [inline resources/views/components/tables/desktop/exchanges.blade.php:19]: Can this use the lang for now?

```suggestion
                <span>(@lang('currencies.usd.currency'))</span>
```

**Review by alexbarnsley — CHANGES_REQUESTED** [review]
quickie

File: resources/views/components/tables/desktop/exchanges.blade.php [hunk 3]
```diff
@@ -7,7 +7,7 @@
             <x-tables.headers.desktop.text name="general.exchange.top_pairs" />
 
             <x-tables.headers.desktop.number name="general.exchange.price">
-                <span>({{ Settings::currency()}})</span>
+                <span>(@lang('currencies.usd.currency'))</span>
```
Comment (alexbarnsley) [inline resources/views/components/tables/desktop/exchanges.blade.php:10]: My bad, it's a config file :hide:

```suggestion
                <span>({{ config('currencies.usd.currency') }})</span>
```

File: resources/views/components/tables/desktop/exchanges.blade.php [hunk 4]
```diff
@@ -16,7 +16,7 @@ class="text-right"
                 breakpoint="md-lg"
                 responsive
             >
-                <span>({{ Settings::currency()}})</span>
+                <span>(@lang('currencies.usd.currency'))</span>
```
Comment (alexbarnsley) [inline resources/views/components/tables/desktop/exchanges.blade.php:19]:
```suggestion
                <span>({{ config('currencies.usd.currency') }})</span>
```

**Review by alexbarnsley — APPROVED** [review]

**Comment by samharperpittam** [comment]
@alfonsobries - PR looks good but just a quick question, were you able to fetch more prices for exchanges than I have here? Just checking my local is setup correctly.

image

**Comment by alfonsobries** [comment]
@samharperpittam For me it loads 7 prices but fixed an issue that was causing the job to fail after some time, you can try again,

Consider that it may take 2 minutes to load all of them because it only tries to load data for 10 exchanges per minute, so, if you keep the queue running, it will run the second batch after 1 minute

**Comment by alexbarnsley** [comment]
>Consider that it may take 2 minutes to load all of them because it only tries to load data for 10 exchanges per minute, so, if you keep the queue running, it will run the second batch after 1 minute

@alfonsobries which command are you running as the `exchanges:load` command doesn't run any jobs :thinking:

**Pull request merged**